distributed-testing/src/main/groovy/net/corda/testing/Tests.java

```diff
@@ -0,0 +1,207 @@
+package net.corda.testing;
+
+import groovy.lang.Tuple2;
+import groovy.lang.Tuple3;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVPrinter;
+import org.apache.commons.csv.CSVRecord;
+import org.jetbrains.annotations.NotNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.io.Writer;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Tests {
```

Comment:
I think JUnit assumes that any class starting or ending with `Test` or `Tests` is a JUnit test class, so this is not a good choice of name.

Reply:
JUnit explicitly searches for the `@Test` annotation now